CmisSync.Lib/Sync/SynchronizedFolder.cs

```diff
@@ -1188,6 +1192,17 @@ private bool DownloadFile(IDocument remoteDocument, string localFolder)
                         if (null != remoteDocument.LastModificationDate)
                         {
                             File.SetLastWriteTime(filepath, (DateTime)remoteDocument.LastModificationDate);
+                        }
+
+                        // Should the local file be made read-only?
+                        // Check ther permissions of the current user to the remote document.
+                        // (eventually set read only after we finish to manipulate the file or we can get an error if it's read only)
+                        bool canSetContent = remoteDocument.AllowableActions.Actions.Contains(PermissionMappingKeys.CanSetContentDocument);
+                        //FIXME: alfresco reporst "canSetContentStream" instead of "canSetContent.Document"

```

Comment:
please, also note this one. Alfresco seem to use "canSetContentStream" instead of "canSetContent.Document"